finetuner/tuner/evaluation.py

```diff
@@ -0,0 +1,83 @@
+from typing import TYPE_CHECKING, Optional, List
+from jina import Document, DocumentArray
+
+from .metrics import METRICS
+from .. import __default_tag_key__
+from ..helper import AnyDNN
+from ..embedding import embed
+
+if TYPE_CHECKING:
+    from ..helper import AnyDNN
+
+
+class Evaluator:
+    def __init__(
+        self,
+        eval_data,
```

Comment:
Some typehinting would be really useful

Reply:
Added `eval_data: 'DocumentSequence'`, `catalog: 'DocumentSequence'`